modules/foodcenter/Classes/FoodcenterPrint.php

```diff
@@ -156,11 +156,8 @@ private function GetDate($time): false|string
     {
         global $func;
 
-        if ($this->config['datetime_format'] == '') {
-            return $func->unixstamp2date($time, "datetime");
-        } else {
-            return $func->unixstamp2date($time, $this->config['datetime_format']);
-        }
+        $datetimeFormat = $this->config['datetime_format'] ?? 'datetime';
+        return $func->unixstamp2date($time, "datetime");
```

Comment:
```suggestion
        return $func->unixstamp2date($time, $datetimeFormat);
```
I think this was your intention?
Also wouldn't `$dateTimeFormat` be the correct(erer) CamelCase naming?

Reply:
Yes, done. Thanks.